src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -371,6 +382,8 @@ def main():
 
     args = parser.parse_args()
 
+    pixel_size = get_pixel_size(args.looks)
```

Comment:
Good practice is to wait to define variables until they're needed to reduce the number of lines a reader needs to keep in their head at once. This is better moved to just before the `translate_outputs()` call on line 414.